env/__init__.py

```diff
@@ -0,0 +1,7 @@
+import gym
+from gym.envs.registration import register
+
+register(
+	id='ObstacleEnv-v0',
+	entry_point='env.envs.gym_locomotion_env:HumanoidBulletEnv'
```

Comment:
This needs to be named something else. Ideally something like ObstacleBulletEnv.